docs/platform/porting/system.md

```diff
@@ -2540,3 +2540,358 @@ Example:
+        "cpu_avg_period": 10,
+    },
+
+    "crash-manager":{
+        "monitor": 0,
+        "apply_to_children": 1,
+    },
+
+    "9zWvGSYU8Z.calendar": {
+        "print_current": 1,
+    }
+}
+```
+
+#### D-Bus signal
+
+When the abnormality is detected, the stability monitor sends one of the following D-Bus signals, depending on the module or parameter that detected or caused the abnormality:
+
+-  **cpu**
+```
+objpath: /Org/Tizen/StabilityMonitor/tsm_cpu
+interface: org.tizen.abnormality.cpu.relative:
+member: "AbnormalityDetected"
+parameters: (isdda{sv})
+    pid <int32>
+    limitType <string>
+    cpuRelative <double>
+    maxAllowedCpuRelative <double>
+    dict {
+        report_path <variant(string)>
+        killed <variant(bool)>
+        process_name <variant(string)>
+        process_state <variant(string)>
+    }
+```
+
+-  **mem**
+```
+objpath:   /Org/Tizen/StabilityMonitor/tsm_mem
+interface: org.tizen.abnormality.mem.relative:
+member:    "AbnormalityDetected"
+parameters: (isdda{sv})
+    pid <int32>
+    limitType <string>
+    memRelative <double>
+    maxAllowedMemRelative <double>
+    dict {
+        report_path <variant(string)>
+        killed <variant(bool)>
+        process_name <variant(string)>
+        process_state <variant(string)>
+    }
+```
+
+-  **io**
+```
+objpath:   /Org/Tizen/StabilityMonitor/tsm_io
+interface: org.tizen.abnormality.io.absolute:
+member:    "AbnormalityDetected"
+parameters: (isdda{sv})
+    pid <int32>
+    limitType <string>
+    IOAbsolute <double>
+    maxAllowedIOAbsolute <double>
+    dict {
+        report_path <variant(string)>
+        killed <variant(bool)>
+        process_name <variant(string)>
+        process_state <variant(string)>
+    }
+```
+
+-  **fd**
+```
+objpath:   /Org/Tizen/StabilityMonitor/tsm_fd
+interface: org.tizen.abnormality.fd.absolute:
+member:    "AbnormalityDetected"
+parameters: (isiia{sv})
+    pid <int32>
+    limitType <string>
+    FDsAbsolute <int32>
+    maxAllowedFDsAbsolute <int32>
+    dict {
+        report_path <variant(string)>
+        killed <variant(bool)>
+        process_name <variant(string)>
+        process_state <variant(string)>
+    }
+```
+
+>  [!NOTE]
+> The following parameters can have a few possible values:
+> -  `limitType` can be either "peak" or "avg",
+> -  `process_name` can either be application ID for applications or executable name for others,
+> -  `process_state` can be either "foreground" or "background".
+
+The following is example signal observed with `dbus-monitor`:
+```
+signal time=1547866746.490843 sender=:1.73054 -> destination=(null destination) serial=3 path=/Org/Tizen/StabilityMonitor/tsm_cpu; interface=org.tizen.abnormality.cpu.relative; member=AbnormalityDetected
+   int32 27124
+   string "peak"
+   double 0.962084
+   double 0.95
+   array [
+      dict entry(
+         string "report_path"
+         variant             string "/opt/usr/share/crash/dump//top_27124_20190118215903.zip"
```

Comment:
Would this path vary?

Can we write like this?

```suggestion
         variant             string "<PATH/FILE_NAME.zip>"

Reply:
The prefix of the path is constant: `/opt/usr/share/crash/`.
I would prefer to leave the path as it is, because this is the example of specific signal observed with dbus-monitor, not a template or so.

Reply:
@kkuchciak thats fine, thanks for clarifying.